webextensions/api/management.json

```diff
@@ -338,10 +338,10 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": false
+                "version_added": "55"
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "55"
               },
```

Comment:
Nit, technically all the management events have been added in Firefox 55 but:
- in Firefox 55, all the events does only support themes (as a confirmation here is `allowedTypes` as landed from [Bug 1336908](https://bugzilla.mozilla.org/show_bug.cgi?id=1336908): https://hg.mozilla.org/mozilla-central/rev/0df833bf2b9f#l3.81)
- in Firefox 56, the read-only part of the API (basically management.get/management.getAll and these events) have been extended to also support extensions (see here the `allowedTypes` as landed from [Bug 1282981](https://bugzilla.mozilla.org/show_bug.cgi?id=1282981): https://hg.mozilla.org/mozilla-central/rev/fe3ed659f568#l2.13)

If we want to capture that in the compatibility table, we could add a `notes` property that points that the API supports the "extension" add-on type starting from Firefox 56.

Reply:
Thanks! I updated this to add a note :)